app/app/templates/data.html

```diff
@@ -1,28 +1,6 @@
-{% extends "base.html" %}{% block title %}ReSQ{% endblock %}{% block navbar %}
-<div class="sidebar-wrapper">
-	<ul class="nav">
-		<li class="nav-item">
-			<a class="nav-link" href="{{ url_for('routes.home')}}">
-				<i class="material-icons">list</i>
-				<p>Queue</p>
-			</a>
-		</li>
-		<li class="nav-item active">
```

Comment:
The only issue I could see was the navbar not updating its color/format depending on what page the user was currently viewing. This was fixed in my latest commit